src/services/urls.ts

```diff
@@ -1,5 +1,5 @@
 export const ImageUrl = 'https://asset-images.messari.io/images/';
 
 export const baseUrl = 'https://data.messari.io/api/v2';
-/*
-https://data.messari.io/api/v2/assets?fields=id,slug,symbol,metrics/market_data&&limit=2 */
+
+export const searchUrl = 'https://data.messari.io/api/v1/assets';
```

Comment:
this should be stored in a .env file

https://yarnpkg.com/package/react-native-dotenv